### ISSUE TITLE
IE - Object Error 

### ISSUE THREAD

**Issue by rmuraliananth** [mattzollinhofer/vue-typeahead-bootstrap#106]
![image](https://user-images.githubusercontent.com/5518060/108429372-dc3c1b00-720d-11eb-9d8c-e7910b853a2b.png)

- Object error occurred in IE 11
- Issue exists in vue-typeahead-bootstrap **version > 2.5.5**

**Comment by mattzollinhofer** [comment]
Yeah, thanks for the report, that looks like a problem that surfaced when we added wider character support (#100). 

I just googled this and found this stackoverflow article: https://stackoverflow.com/questions/54242799/ie11-object-doesnt-support-property-or-method-normalize

It suggests polyfilling the `normalize` method. I don't know the best way to do that for a component like this one off the top of my head so I'll have to come back to this one day.

I would love any help looking into this. @aeyoll, @eduardosada (or anyone else) have any time to take a look into this?

**Comment by aeyoll** [comment]
@mattzollinhofer @rmuraliananth Looks like the best polyfill for this is "unorm" (https://github.com/walling/unorm), as suggested in the core-js repo (https://github.com/zloirock/core-js#missing-polyfills).
